packages/components/src/components/wizard/wizard.tsx

```diff
@@ -23,10 +23,9 @@ const Wizard = ({
     const getSteps = React.useCallback(() => React.Children.toArray(children), [children]);
 
     React.useEffect(() => {
-        const local_initial_step = initial_step - 1;
         const local_children = getSteps();
 
-        if (local_initial_step && local_children[local_initial_step]) {
+        if (initial_step > 0 && local_children[initial_step]) {
```

Comment:
Any reason for this change?
is initial step always going to be 0, wouldn't any other value be passed to it?
Like before it was decrementing by 1 now it will be as it is.
Just curious

Reply:
@hamza-deriv you can see previous `initial_step` was 1 by default, but `local_initial_step = initial_step - 1`. In result when I pass initial_step to 1, in result setActiveStep didn't work, because `local_initial_step === 0` and it's falthy. In result we can't set `initial_step` to 1, just for 2 and more.  But I needed to control it... Now i'm checking this value MORE than 0 and only after this set step by `setActiveStep`. Hope, you understand what I mean :)

Reply:
But my question is that if initial step is 2 then the condition be like 
`if (2 > 0 && local_children[2]) { `

meanwhile the step value before was 
`if (2 > 0 && local_children[2-1]) {`

Hope you get what i am asking 😄

Reply:
@hamza-deriv if the init_step = 1, in result local_initial_step = 0, and in result we are not going inside of the condition and don't set step to ONE, active_step will be ZERO by default

Reply:
@hamza-deriv I could just write

if (local_initial_step `> 0` && local_children[local_initial_step]) {...

But made decision to fully delete local_initial_step because we don't need it at all in the component and use it inside of this component